app/src/main/java/io/gnosis/safe/utils/ViewUtils.kt

```diff
@@ -0,0 +1,32 @@
+package io.gnosis.safe.utils
+
+import android.annotation.SuppressLint
+import android.view.MotionEvent
+import android.view.View
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.Toast
+
+enum class DrawablePosition(val index: Int) {
+    DRAWABLE_LEFT(0),
+    DRAWABLE_TOP(1),
+    DRAWABLE_RIGHT(2),
+    DRAWABLE_BOTTOM(3)
+}
+
+@SuppressLint("ClickableViewAccessibility")
+inline fun TextView.setOnCompoundDrawableClicked(
```

Comment:
Is this used?

Reply:
Not anymore. In the address inputs the clicks were confined the the compound drawable but now I made the entire view clickable as it is no longer editable by hand anyhow. I will remove this as well in the cleanup tasks that I mentioned earlier.